cli/foundation-models/system/finetune/image-classification/multilabel-classification/hftransformers-fridgeobjects-multilabel-classification-pipeline.yaml

```diff
@@ -58,10 +58,12 @@ jobs:
       training_data: ${{parent.inputs.training_data}}
       validation_data: ${{parent.inputs.validation_data}}
 
-      image_width: 224
-      image_height: 224
-      number_of_workers: 8
+      auto_hyperparameter_selection: False
```

Comment:
aren't we going to remove the "auto_hyperparameter_selection" parameter?

Reply:
Will happen in subsequent release. @shubhamiit is working on the changes.